04-single-hop-6lowpan-icmp/example_test_guide.md

```diff
@@ -0,0 +1,168 @@
+_The following is a possible way to run the tests, this is just an example_
+## Goal: Check single hop 6lowpan icmp
+
+_note: Since the ping6 function is now async some interval times may need to
+be increased to compensate_
+
+### Pre-setup Steps
+1. Get 2 of the following boards: `samr21-xpro`, `iotlab-m3`
+2. Get the following boards:`remote-revb`, `arduino-zero` with `xbee` module
```

Comment:
Add hint about IoT-LAB? All except the `remote-revb` are provided there.

Reply:
Will do.